src/account/accountReducers.js

```diff
@@ -58,7 +59,7 @@ export default (state: AccountState = initialState, action: Action) => {
 
       const { type, ...newAccount } = action; // eslint-disable-line no-unused-vars
 
-      if (accountIndex === -1) {
+      if (accountIndex === NULL_ACCOUNT_INDEX) {
```

Comment:
I don't think this is a good idea. -1 is a well known 'not known' value, while we add nothing by adding a constant here.